lib/vsc/utils/optcomplete.py

```diff
@@ -105,7 +105,7 @@
 from optparse import OptionParser, Option
```

Comment:
i'm ok with dropping this module, it's unused because too much latency

Reply:
it was easy enough to fix the broken tests, so I left it in

if we really want to remove it, we can follow up on that...